src/index.html

```diff
@@ -22,6 +22,323 @@
+        </div>
+        <div class="card__cost">
+          <p class="card__price">Price:</p>
+          <p class="card__total">$2,199</p>
+        </div>
+        <button
+          class="card__button"
+          data-qa="card-hover"
+        >
+          Buy
+        </button>
+      </div>
+
+      <div class="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="iMac"
+          class="card__image"
+        />
+        <h2 class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__stars">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+        <div class="card__cost">
+          <p class="card__price">Price:</p>
+          <p class="card__total">$2,199</p>
+        </div>
+        <button class="card__button">Buy</button>
+      </div>
+
+      <div class="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="iMac"
+          class="card__image"
+        />
+        <h2 class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__stars">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+        <div class="card__cost">
+          <p class="card__price">Price:</p>
+          <p class="card__total">$2,199</p>
+        </div>
+        <button class="card__button">Buy</button>
+      </div>
+
+      <div class="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="iMac"
+          class="card__image"
+        />
+        <h2 class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__stars">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+        <div class="card__cost">
+          <p class="card__price">Price:</p>
+          <p class="card__total">$2,199</p>
+        </div>
+        <button class="card__button">Buy</button>
```

Comment:
This product card is identical to the first one. Consider varying the content to display different products.